handlers/course.py

```diff
@@ -12,6 +14,7 @@
 
 
 @router.get("/courses/", response_model=List[CourseListOut], tags=["Admin dashboard"])
-async def read_courses_list(auth = Depends(get_current_user)):
-    query = course.select()
+# async def read_courses_list(auth=Depends(get_current_user)):
```

Comment:
Why do you comment these lines?
Anyway, it seems to be unintended change. Search for string `# async def` to find all similar comments.

BTW, you can use `git commit --amend` or interactive rebase to "update" existing commit instead of adding new one

Reply:
Now, I'm looking in code, and I see  - all clear. I think, it was bad commit, I shouldn't have done it. My bad. 
But I temporary cleaned from auth for more comfortable debugging.
Next time I will be more attentive in commits

Reply:
It was fixed in a new commit https://github.com/anton-serozhechkin/e_university_api/pull/51/commits/45e88306755e04c7bbd02e5bc3a1ab0bcadd68a7